packages/app/ios/RNFBApp/RNFBSharedUtils.m

```diff
@@ -164,4 +164,40 @@ + (BOOL)getConfigBooleanValue:(NSString *)tag key:(NSString *)key defaultValue:(
   return enabled;
 }
 
+/**
+ * Recursively converts sentinel objects back to NSNull.
+ * This works around iOS TurboModule's limitation where null values in object properties get stripped.
+ * The JavaScript side converts nulls to { __rnfbNull: true } sentinel objects,
```

Comment:
going to make a suggestion to constant-ify this, so remove the specific value here

```suggestion
 * The JavaScript side converts nulls to sentinel objects,
```